R/calcPOP.R

```diff
@@ -1,9 +1,22 @@
-calcPOP <- function(scenario = 'SSP2') {
+#' calcPOP
+#'
+#' The SSP data filtered by pop
+#'
+#' @param scenario string. By choosing a scenario you filter the SSP dataset
```

Comment:
better list which values the argument `scenario` can take ("SSP1", "SSP2", etc.) and phrase the argument description like this: "SSP scenario choice"

Reply:
I itemize the scenario, is it ok?

Reply:
perfect, thanks